src/edu/washington/escience/myria/HdfsTupleWriter.java

```diff
@@ -0,0 +1,64 @@
+package edu.washington.escience.myria;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+import org.apache.hadoop.fs.FSDataOutputStream;
+
+import edu.washington.escience.myria.storage.ReadableTable;
+
+/**
+ * HdfsTupleWriter is a {@link TupleWriter} that writes tuples to HDFS.
+ */
+public class HdfsTupleWriter implements TupleWriter {
```

Comment:
Could we rename this class according to the format it writes rather than its expected destination (which has nothing to do with the implementation of this class)? Even something like "MyriaCustomPersistenceFormatTupleWriter" is fine.